src/man/firejail.1.in

```diff
@@ -96,6 +96,13 @@ $ firejail [OPTIONS] firefox        # starting Mozilla Firefox
 .PP
 # sudo firejail [OPTIONS] /etc/init.d/nginx start
 
+When an option is specified multiple times (whether in a profile, on the
+command line, or both) or conflicts with a related option, the
+precedence/behavior is option-specific and usually documented in the
+\fBOPTIONS\fR section below. Note that any option specified in a profile can be
+disabled on the command line using \fB--ignore\fR.
```

Comment:
FTR: `--profile=firefox --ignore='net none'` does not work. It works if you flip the order. Nevermind I think it makes things to complicated to explain every detail here.

Reply:
I've made the statement a bit weaker by adding "generally", so a more detailed explanation can be given in `--ignore``'s own section.